### DISCUSSION TITLE
How to use @vercel/og without NextJS

### DISCUSSION THREAD

**Discussion by DaPotatoMan** [vercel/vercel#11001] [Help] [answered]
### Question

I'm sorry if this is not the right place to ask. I have been struggling to use @vercel/og with my vite app.
I tried the official example and used api/og.jsx and api/og.tsx but neither of those work.
I also tried transpiling the jsx file with tsup but no luck as well.

Is there any official guide to use it without NextJS?

**Comment by josiahwiebe** [comment, marked as answer]
This is by no means an official guide, but here's how I was able to get it work: https://jwie.be/blog/using-vercel-og-without-nextjs
